packages/backend/src/api/endpoints/SamlConnectionApi.ts

```diff
@@ -0,0 +1,90 @@
+import type { SamlIdpSlug } from '@clerk/types';
+
+import { joinPaths } from '../../util/path';
+import type { SamlConnection } from '../resources';
+import { AbstractAPI } from './AbstractApi';
+
+const basePath = '/saml_connections';
+
+type SamlConnectionListParams = {
+  limit?: number;
+  offset?: number;
+};
+type CreateSamlConnectionParams = {
+  name: string;
+  provider: SamlIdpSlug;
+  domain: string;
+  idpEntityId?: string;
+  idpSsoUrl?: string;
+  idpCertificate?: string;
+  idpMetadataUrl?: string;
```

Comment:
🔧 We are missing the optional `idpMetadata` which is actually the metadata file content